Browser/keywords/getters.py

```diff
@@ -558,15 +558,14 @@ def get_selected_options(
         logger.info(response)
         expected = list(assertion_expected)
         selected: Union[List[int], List[str]]
-        if option_attribute is SelectAttribute.value:
-            selected = [sel.value for sel in response.entry if sel.selected]
-        elif option_attribute is SelectAttribute.label:
-            selected = [sel.label for sel in response.entry if sel.selected]
-        elif option_attribute is SelectAttribute.index:
-            selected = [
-                index for index, sel in enumerate(response.entry) if sel.selected
-            ]
+
+        if option_attribute is SelectAttribute.index:
```

Comment:
Instead of `is` should we use `==` in this case?

Reply:
https://docs.python.org/3/library/enum.html#comparisons

as far as i understood it does not make any difference.
And in that case i do like the is more than the ==.
Because that shows that this is really not comparing a value of a variable to be equal but expecting a specific identity of enum member.